package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "chm-components",
+  "description": "chm-components",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": "true",
+  "readme": "https://github.com/scbd/chm-components",
+  "author": "Secretariat of the Convention on Biological Diversity <it@cbd.int>",
+  "homepage": "https://github.com/scbd/chm-components",
+  "workspaces": {
+    "packages": [
+      "packages/components/*"
```

Comment:
just packages/8 not  packages/.....  Not sure if * is even needed.

Reply:
Done! For now I used *.